src/components/UserProfile/ProfileInput/ProfileInput.js

```diff
@@ -1,24 +1,35 @@
-import React from 'react'
+import React, {useState} from 'react'
 import "./ProfileInput.css";
 
 
 function ProfileInput(props) {
-    const {label, name, value, handleChange, type, placeholder} = props
+    const {label, name, value, handleChange, type, placeholder, errorMessage, ...inputProps} = props
+    const [focused, setFocused] = useState(false)
 
+    function handleFocus(e) {
+        setFocused(true)
+    }
     return (
         <div className="profile-input">
             <div className="profile-input-label">
                 <label className="profile-name">{label}</label>
             </div>
             <div className="profile-input-field-container">
-                <input
-                    className="profile-input-field"
-                    name={name}
-                    type={type}
-                    value={value}
-                    onChange={handleChange}
-                    placeholder={placeholder}
+                <input {...inputProps}
+                       className="profile-input-field"
+                       name={name}
+                       type={type}
+                       value={value}
+                       onChange={handleChange}
+                       placeholder={placeholder}
+                       onBlur={handleFocus}
+                       onFocus={() =>
+                           name === "username" && setFocused(true)
+                       }
+                       disabled={name === "username"}
+                       focused={focused.toString()}
                 />
+                <span className="profile-error-text"> {errorMessage}</span>
```

Comment:
I was testing out this modal and it wasn't centered could you please restyle it to center.